### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by ZD96** [new92/igfi#33] [Q&A] [answered]
Hi, i did few test with V1, 2 and 3 and i always got this error, am i doing something wrong? 
![Screenshot 2023-08-24 155901](https://github.com/new92/IGFollowersIncreaser/assets/138192230/b05d7994-efb8-463b-b46f-8a055e1e78b0)

**Comment by new92** [comment, marked as answer]
Hey @ZD96, no you're not doing something wrong but the error you're getting is indicating that you should stop temporarily using the script and start using it again after a couple of hours.
